test/web-auth/hosted-pages.test.js

```diff
@@ -386,4 +386,67 @@ describe('auth0.WebAuth._universalLogin', function() {
       );
     });
   });
+
+  context('getSSOData', function() {
+    before(function() {
+      this.auth0 = new WebAuth({
+        domain: 'me.auth0.com',
+        clientID: '...',
+        redirectUri: 'http://page.com/callback',
+        responseType: 'code',
+        _sendTelemetry: false
+      });
+    });
+
+    afterEach(function() {
+      request.get.restore();
+    });
+
+    it('should call ssodata with all the options', function(done) {
+      stub(request, 'get', function(url) {
+        expect(url).to.be('https://me.auth0.com/user/ssodata/');
+        return new RequestMock({
+          headers: {},
+          cb: function(cb) {
+            cb(null, {
+              body: {
+                sso: false
+              }
+            });
+          }
+        });
+      });
+
+      this.auth0._universalLogin.getSSOData(function(err, data) {
+        expect(err).to.be(null);
+        expect(data).to.eql({
+          sso: false
+        });
+        done();
+      });
+    });
+    it('should call ssodata with all the ad options', function(done) {
```

Comment:
ssodata -> ssoData. Also I'd call it: "should call ssoData with all the options including activeDirectories/ad"

Reply:
I made AD uppercase